seeds/02_scope.ts

```diff
@@ -49,6 +52,7 @@ const scopes = [
   }
 ]
 
-export function seed (knex: Knex): Knex.QueryBuilder<number[]> {
-  return knex('Scope').insert(scopes)
+export function seed (knex: Knex): Promise<Knex.QueryBuilder<number[]>> {
+  return knex(tableName).del()
```

Comment:
let keep consistency and do not keep the table name in the external variable but use the string value instead `Scope` as you are doing it in the line after

Reply:
agreed